libphal/phal_sbe.C

```diff
@@ -29,6 +29,8 @@ using namespace openpower::phal::pdbg;
 constexpr uint16_t minFFDCPackageWordSizePOZ = 5;
 constexpr uint16_t pozFfdcMagicCode = 0xFBAD;
 constexpr uint16_t slidOffset = 2 * sizeof(uint32_t);
+constexpr bool CO_CMD_SUCCESS = 1;
```

Comment:
Use `true` or `false` rather than `0` or `1` (although they mean same but for better conformation to the standard)